topics/crystal/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: crystal-lang
+created_by: Ary Borenszweig
+display_name: Crystal
+logo: crystal.png
+released: June 2011
+short_description: Crystal is a self-hosted, general-purpose programming language.
+topic: crystal
+url: https://crystal-lang.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Crystal_(programming_language)
+---
+Crystal is a programming language with friendly syntax, static type checking and modern stdlib. It compiles into efficient, easy to distribute native code.
```

Comment:
`easy to distribute` -> `easy-to-distribute`

Reply:
Done! 👍